ush/forecast_predet.sh

```diff
@@ -568,7 +568,13 @@
 
   # CICE does not have a concept of high frequency output like FV3
   # Convert output settings into an explicit list for CICE
-  CICE_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "${FHOUT_OCNICE}" "${FHMAX}")
+  if (( $(( ( cyc + FHMIN ) % FHOUT_ICE )) == 0 )); then
+    mapfile -t -d ' ' CICE_OUTPUT_FH < <(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}") || exit 10
```

Comment:
Try this to fix the extra new-line
```suggestion
    mapfile -t CICE_OUTPUT_FH < <(seq "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}" || exit 10
```

Reply:
Repeat with the next `mapfile`

Reply:
Thank you @WalterKolczynski-NOAA. After a quick test, this seems to have fixed the extra new-line.

Reply:
A little bit what's going on:
We were specifying a delimiter for `seq` (a hold-over from earlier when not using `mapfile`), but `seq` is still going to tack a new line at the end of the output. `mapfile` was then using the same delimiter (space), which meant the `-t` would try to remove an extra space from the end instead of a new line. Now we are using the default delimiter (new line) for both, so `-t` works as we wanted it to.